ioctl/cmd/hdwallet/hdwalletcreate.go

```diff
@@ -82,6 +85,62 @@ func hdwalletCreate() error {
 
 	output.PrintResult(fmt.Sprintf("Mnemonic pharse: %s\n"+
 		"It is used to recover your wallet in case you forgot the password. Write them down and store it in a safe place.", mnemonic))
-
 	return nil
 }
+
+// DeriveKey derives the key according to path
+func DeriveKey(account, change, index uint32, password string) (string, crypto.PrivateKey, error) {
+	// derive key as "m/44'/304'/account'/change/index"
+	hdWalletConfigFile := config.ReadConfig.Wallet + "/hdwallet"
+	if !fileutil.FileExists(hdWalletConfigFile) {
+		return "", nil, output.NewError(output.InputError, "Run 'ioctl hdwallet create' to create your HDWallet first.", nil)
+	}
+
+	enctxt, err := ioutil.ReadFile(hdWalletConfigFile)
+	if err != nil {
+		return "", nil, output.NewError(output.InputError, "failed to read config", err)
+	}
+
+	enckey := util.HashSHA256([]byte(password))
+	dectxt, err := util.Decrypt(enctxt, enckey)
+	if err != nil {
+		return "", nil, output.NewError(output.InputError, "failed to decrypt", err)
+	}
+
+	dectxtLen := len(dectxt)
+	if dectxtLen <= 32 {
+		return "", nil, output.NewError(output.ValidationError, "incorrect data", nil)
+	}
+
+	mnemonic, hash := dectxt[:dectxtLen-32], dectxt[dectxtLen-32:]
+	if !bytes.Equal(hash, util.HashSHA256(mnemonic)) {
+		return "", nil, output.NewError(output.ValidationError, "password error", nil)
+	}
+
+	wallet, err := hdwallet.NewFromMnemonic(string(mnemonic))
+	if err != nil {
+		return "", nil, err
+	}
+
+	derivationPath := fmt.Sprintf("m/44'/304'/%d'/%d/%d", account, change, index)
```

Comment:
should use `DefaultRootDerivationPath` here, also i think it should be easier to define:
```go
const DefaultRootDerivationPath = "m/44'/304'"
```
and Sprintf "/a/b/c" after it